spec/features/messages_feature_spec.rb

```diff
@@ -0,0 +1,29 @@
+feature 'messages' do
```

Comment:
There is probably a small refactor you could do to these tests to keep them DRY. Some of the set up steps are very similar across them and could be extracted to a helper method.  It's a very small thing though.